homework-submission/homework-week1/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+    <style>
+    .itemCompleted {
+        text-decoration: line-through;
+      }
+      .itemNotCompleted {
+        text-decoration: none;
+      }
+    </style></style>
+  </head>
+  <body>
+    <div id="thisIsWhereReactIsInjected"></div>
```

Comment:
When fallowing naming conventions the id is better called root.

Reply:
okay